test/e2e/serial/tests/tolerations.go

```diff
@@ -784,10 +830,63 @@ func sriovToleration() corev1.Toleration {
 	}
 }
 
-func waitForMcpUpdate(cli client.Client, ctx context.Context, mcps []*machineconfigv1.MachineConfigPool) {
-	By("waiting for mcp to start updating")
-	waitForMcpsCondition(cli, ctx, mcps, machineconfigv1.MachineConfigPoolUpdating)
+func waitForMcpUpdate(cli client.Client, ctx context.Context, mcpsInfo []mcpInfo, updateType MCPUpdateType) {
+	for _, info := range mcpsInfo {
+		/*
+				For every mcp check the following:
+				1. soft requirement: loop over until condition Updating==true
+			    2. loop over until condition Updated==true, is a must
+				3. check the sample node is updated with new config in its annotations, both for desired and current, is a must
+		*/
```

Comment:
good. Better:
1. fix spacing, it is unnecessarily misaligned and harder to read than it can
2. consistency. The first bullet starts with required/desired, the second and third bullet but the requirement at the end.
Example of how it can read:
```
/*
    procedure:
    1. soft requirement: loop over until condition Updating==true
    2. hard requirement: loop over until condition Updated==true
    3. hard requirement: check the sample node is updated with new config in its annotations, both for desired and current
*/
```